core/src/main/resources/hudson/model/AllView/noJob.jelly

```diff
@@ -32,7 +32,7 @@ THE SOFTWARE.
   <j:choose xmlns:j="jelly:core">
     <!-- Only show the create new jobs link to those users that have permission to use it. -->
     <j:getStatic var="permission" className="hudson.model.Item" field="CREATE"/>
-    <j:when test="${h.hasPermission(permission)}">
+    <j:when test="${h.hasPermission(it.owner, permission)}">
```

Comment:
Another old bug that is more visible if we are advocating use of folders: this link was supposed to appear based on Job/Create permission on the folder, not necessarily root level.